zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/FormHandler.java

```diff
@@ -79,7 +81,13 @@ public void updateEntity(final Record<Form> record, final FormEntity entity) {
 
   @Override
   public void flush(final FormEntity entity, final BatchRequest batchRequest) {
-    batchRequest.add(indexName, entity);
```

Comment:
❓ Not clear to me why isDeleted was not published before?

Reply:
It was published before, but it created a new record marked as `deleted` with a version increment.